src/site/content/en/fast/identify-slow-third-party-javascript/index.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title: Identify slow third-party JavaScript
+subhead: Supercharge your performance detective skills with Lighthouse and Chrome DevTools.
+authors:
+  - mihajlija
+date: 2019-07-16
+# Add an updated date to your post if you edit in the future.
+# updated: 2019-06-27
+hero: hero.jpg
+alt: Inspecting a laptop with a magnifying glass.
+# You can adjust the position of your hero image with this property.
+# Values: top | bottom | center (default)
+# hero_position: bottom
+description: |
+  Learn how to use Lighthouse and Chrome DevTools to identify slow third-party resources.
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - fast
+---
+
+<style>
+  img[src*="icon"] {
+    max-width: 16px;
+    vertical-align: middle;
+  }
+</style>
+
+As a developer, you often don't have control over [which third-party scripts](/third-party-javascript/#network) your site loads. Before you can optimize third-party content you have to do some detective work to find out what's making your site slow. 🕵️
+
+In this post, you'll learn how to use [Lighthouse](https://developers.google.com/web/tools/lighthouse/) and [Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools/) to identify slow third-party resources. The post walks through increasingly robust techniques which are best used in combination.
+
+## If you only have 5 minutes
+
+The Lighthouse [Performance audit](/lighthouse-performance) helps you discover opportunities to speed up page loads. Slow third-party scripts are likely to appear in the **Diagnostics** section under the "Reduce JavaScript execution time" and "Avoid enormous network payloads" audits.
```

Comment:
@mfriesenhahn I sometimes suggest folks bold the names of audits instead of putting them in quotes. What do you prefer?

Reply:
Went back and forth on this myself. If you prefer bold, that's cool with me.

Reply:
Yeah let's go with bold